motionplan/rrtStarConnect.go

```diff
@@ -119,7 +119,12 @@ func (mp *rrtStarConnectMotionPlanner) rrtBackgroundRunner(ctx context.Context,
 		}
 		rrt.maps = planSeed.maps
 	}
-	target := newConfigurationNode(referenceframe.InterpolateInputs(seed, rrt.maps.optNode.Q(), 0.5))
+	targetConf, err := mp.frame.Interpolate(seed, rrt.maps.optNode.Q(), 0.5)
+	target := newConfigurationNode(targetConf)
+	if err != nil {
```

Comment:
[minor]: I think we can move this error check up one line

Reply:
:+1: